src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/RedundantImportCheck.java

```diff
@@ -48,6 +48,17 @@
  * &lt;module name="RedundantImport"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * package Test; // OK
+ * import Test.MyClass; // violation, imported from the same package as the current package
```

Comment:
Please add two static imports with 'java.lang' package and same package, like:
```java
import static java.lang.Integer.MAX_VALUE; // ok, static import
import static Test.MyClass.*; // ok, static import


```

Reply:
Done @nmancus1

Reply:
Please organize imports like https://checkstyle.sourceforge.io/idea.html#Organize_Imports (static at the top) or at least separate static and non-static imports to make it easy for reader.

Reply:
```
package Test; // OK
import static Test.MyClass.*; // OK, static import
import static java.lang.Integer.MAX_VALUE; // OK, static import

import Test.MyClass; // violation, imported from the same package as the current package
import java.lang.String; // violation, the class imported is from the 'java.lang' package
import java.util.Scanner; // OK
import java.util.Scanner; // violation, it is a duplicate of another import
public class MyClass{ };
```
Like this @nmancus1 ?

Reply:
Yes, I think this is much easier to read.

Reply:
Done @nmancus1

Reply:
Resolved.